cmd/cmd.go

```diff
@@ -66,7 +66,7 @@ func init() {
 	// will be global for your application.
 
 	rootCmd.PersistentFlags().StringVar(&kubeConfig, "kubeconfig", "", "kubernetes config path")
-	rootCmd.PersistentFlags().StringVar(&sourceStorageClass, "sourcestoraageclass", "", "source storageclass from which all PVC need to be migrated")
+	rootCmd.PersistentFlags().StringVar(&sourceStorageClass, "sourcestorageclass", "", "source storageclass from which all PVC need to be migrated")
 	rootCmd.PersistentFlags().StringVar(&destinationStorageClass, "destinationstorageclass", "", "destination storageclass (CSI storageclass) to which all PVC need to be migrated")
```

Comment:
Maybe these should be a separate PR, but seems good for consistency
```suggestion
	rootCmd.PersistentFlags().StringVar(&destinationStorageClass, "destination-storageclass", "", "destination storageclass (CSI storageclass) to which all PVC need to be migrated")
```